fasthttp/README.md

```diff
@@ -0,0 +1,139 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry fasthttp Handler for Sentry-go SDK
+
+**Godoc:** https://godoc.org/github.com/getsentry/sentry-go/fasthttp
+
+**Example:** https://github.com/getsentry/sentry-go/tree/master/example/fasthttp
+
+## Installation
+
+```sh
+go get github.com/getsentry/sentry-go/fasthttp
+```
+
+```go
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/getsentry/sentry-go"
+	sentryfasthttp "github.com/getsentry/sentry-go/fasthttp"
```

Comment:
If not imported as `sentryfasthttp`, it should primitively be `fasthttp`.

Reply:
`fasthttp` is the name of the package we wrap, so I cannot use the same name here. `goimports` takes this name automatically from the package's top declaration.